packages/shared-lib/src/core/deviceConfigManifest.ts

```diff
@@ -11,179 +11,59 @@
  * describe some properties to be rendered inside this table
  */
 
+import { TSRActionSchema } from 'timeline-state-resolver-types'
 import { TranslationsBundle } from '../lib/translations'
 
 export interface DeviceConfigManifest {
 	/**
 	 * A description of the config fields
 	 */
-	deviceConfig: ConfigManifestEntry[]
+	deviceConfigSchema: string
```

Comment:
Shouldn't this be `JSONBlob<JSONSchema>`?

Reply:
Good idea, I forget we have that.

It does make a bit of a mess in the gateways to translate things across, but I think it is worth it